disasm-test/Main.hs

```diff
@@ -476,7 +480,8 @@ parseBC pfx bc = do
     when dets $ liftIO $ do
       -- Informationally display if there are differences between the llvm-dis
       -- and llvm-disasm outputs, but no error if they differ.
-      ignore (Proc.callProcess "diff" ["-u", norm, parsed])
+      putStrLn "## Output differences: LLVM's llvm-dis <--> this llvm-disasm"
+      ignore (Proc.callProcess "diff" ["-u", "-b", "-B", "-w", norm, parsed])
```

Comment:
This passes different flags to `diff` as compared to its other use in `diffCmp`. Should these flags be the same in both call sites?

Reply:
This diff is only displayed to the user (when the new `--details` flag is specified) and isn't used for any actual success or failure determination.  The other diff *is* part of that determination; perhaps it should be tolerant of whitespace changes, but that would need to be pretty carefully evaluated, whereas my goal here was  to allow the user to focus on significant changes rather than the whitespace "noise".

Reply:
I'll leave it to you to decide how this `diff` should behave, but I would recommend commenting why this `diff` ignores whitespace, since this is seemingly an intentional difference from how `diffCmp` works.